samples/package.json

```diff
@@ -18,8 +18,8 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
+    "gts": "^3.0.2",
```

Comment:
So generally we don't include `gts` in the `samples` folder, and rely on it being installed at the top level package.

Reply:
Done. Good to know!